packages/director/src/app.ts

```diff
@@ -101,12 +115,18 @@ router.post(
   catchRequestHandlerErrors(async (req, res) => {
     const executionDriver = await getExecutionDriver();
     if (executionDriver.id !== 'in-memory')
-      return res.status(405).send('This is only available for in-memory db. Please use the dashboard to set your hooks.');
+      return res
```

Comment:
This is only a change because the settings from prettierrc.js were applied correctly.